bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx96-de.html

```diff
@@ -0,0 +1,103 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "macros-protocol.html" import card with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}Was ist neu in Firefox? – Mehr Privatsphäre, mehr Schutz.{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}Was ist neu in Firefox? – Mehr Privatsphäre, mehr Schutz.{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_96_de') }}
+{% endblock %}
+
+{% set utm_campaign = campaign + '-de' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <div class="show-fxa-supported-signed-out show-fxa-default show-fxa-not-fx show-fxa-ios show-fxa-android">
+        <div class="logo-lockup">
+          <img src="{{ static('protocol/img/logos/firefox/browser/logo.svg') }}" width="64" height="64" alt="Firefox"> <span>+</span> <img src="{{ static('protocol/img/logos/firefox/relay/logo-white.svg') }}" width="64" height="64" alt="Relay">
+        </div>
+        <h1 class="wnp-main-title">Ein Firefox Konto ist eine hervorragende Idee</h1>
+        <p class="wnp-main-tagline">
+          Mehr als nur ein Konto – mit einem Firefox Account stärkst du deine Privatsphäre nachhaltig. Profitiere von kostenlosen Tools wie Firefox Relay und schütze deine private E-Mail Adresse effektiv vor Spam und unkontrollierter Weitergabe an Dritte.
+        </p>
+
+        <p class="wnp-main-cta">
+          <a href="https://relay.firefox.com/?utm_source=mozilla.org-whatsnew96&utm_medium=referral&utm_campaign=whatsnew96-de" class="mzp-c-button mzp-t-product" data-cta-type="button" data-cta-text="Sign up for Relay">Bei Relay anmelden</a>
+        </p>
+      </div>
+
+      <div class="show-fxa-supported-signed-in">
+        <img class="wnp-main-logo" src="{{ static('protocol/img/logos/firefox/relay/logo-white.svg') }}" width="64" height="64" alt="">
+        <h1 class="wnp-main-title">Schütze deine E-Mail mit Firefox Relay</h1>
+        <p class="wnp-main-tagline">
+          E-Mail-Adressen müssen ständig im Web angegeben werden. Das ist auch OK, allerdings nur bei Seiten, denen man zu 100% vertraut. Firefox Relay - Teil deines Firefox Kontos - erstellt Aliasse, die du anstelle deiner privaten E-Mail überall da angeben kannst, wo du deine private lieber für dich behältst.
```

Comment:
**nit:** Referencing [the copy for this ticket](https://docs.google.com/document/d/1bJISjjszNg8Zhvl15pl2irb1UFjkuCqnLhUpuyrp3cs/edit) I think this body paragraph is missing `Wir müssen unsere` in the beginning?

Reply:
Thanks, yes copy is not yet final (I hadn't expected someone to review this yet, but thanks for the heads up regardless).